frontend/ui/src/components/kanban-board/task-card.tsx

```
@@ -7,108 +1,293 @@
+        )}
+        <div className="flex items-center justify-between gap-2 pb-2 pt-6">
+          <div className="flex items-center gap-4">
+            {card.filesData && (
+              <>
+                <button className="flex items-center gap-1">
+                  <Paperclip size={20} />
+                  <span>{card.filesData.length}</span>
+                </button>
+              </>
+            )}
+            <button className="flex items-center gap-1">
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)